index.html

```diff
@@ -44,24 +48,250 @@ <h2 class="primary-header">Glad to see you!</h2>
+      </div>
+    </section>
+    <section class="wrapper card">
+      <div class="img-container">
+        <img width="600" height="400" src="./src/images/facebook.webp" alt="">
+      </div>
+      <div class="description">
+        <h2 class="secondary-header">
+          Facebook 360
+        </h2>
+        <ul class="counter">
+          <li class="active">
+            <p>FACEBOOK</p>
+          </li>
+          <li>
+            <img width="5" height="5" src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>Full Stack Dev</p>
+          </li>
+          <li>
+            <img width="5" height="5" src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>2015</p>
+          </li>
+        </ul>
+        <p class="details">
+          Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360
+          photos and videos on Gear VR.
+        </p>
+        <ul class="tags">
+          <li class="tag-item">html</li>
+          <li class="tag-item">Ruby on rails</li>
+          <li class="tag-item">css</li>
+          <li class="tag-item">javaScript</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </section>
+    <section class="wrapper card">
+      <div class="description">
+        <h2 class="secondary-header">
+          Uber Navigation
+        </h2>
+        <ul class="counter">
+          <li class="active">
+            <p>Uber</p>
+          </li>
+          <li>
+            <img width="5" height="5" src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>Lead Developer</p>
+          </li>
+          <li>
+            <img width="5" height="5" src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>2018</p>
+          </li>
+        </ul>
+        <p class="details">
+          A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer:
+          your car.
+        </p>
+        <ul class="tags">
+          <li class="tag-item">html</li>
+          <li class="tag-item">Ruby on rails</li>
+          <li class="tag-item">css</li>
+          <li class="tag-item">javaScript</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+      <div class="img-container">
+        <img width="600" height="400" src="./src/images/uber.webp" alt="">
+      </div>
+    </section>
+    <!-- End card section -->
+    <!-- Start About me section -->
+    <section class="about-me">
+      <div class="wrapper about-me-flex">
+        <div class="about-intro">
+          <h2 class="secondary-header exception-header">About <br>Myself</h2>
+          <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look
```

Comment:
Great design @MahdiSohaily 🤝

- [ ] [Optional] Kindly fix the `margin bottom` of your design. A big gap is present in your design, which makes it differ from the one provided in the Figma template.
![gap](https://user-images.githubusercontent.com/63917725/185635027-6d7f79dc-ff49-4454-83b9-db49bc1f146d.PNG)

Reply:
I have created this space because it's going to be overlapped by contact form